client/src/app/services/auth.service.ts

```diff
@@ -0,0 +1,18 @@
+import {Injectable} from "@angular/core";
+import {HttpClient} from "@angular/common/http";
+
+@Injectable()
+export class AuthService {
+  _loginApi = 'http://localhost:5200/api/auth/login'
+  _registerApi = 'http://localhost:5200/api/auth/register'
```

Comment:
если делаем локальными указывая префикс _ то напиши что это приватный метод
```suggestion
  private _loginApi = 'http://localhost:5200/api/auth/login'
  private _registerApi = 'http://localhost:5200/api/auth/register'
```